dex/src/main/scala/com/wavesplatform/dex/tool/connectors/AuthServiceRestConnector.scala

```diff
@@ -38,19 +30,21 @@ case class AuthServiceRestConnector(target: String, chainId: Byte) extends RestC
 
   def getAuthCredentials(maybeSeed: Option[String]): ErrorOr[AuthCredentials] = {
 
-    val keyPair       = KeyPair(crypto secureHash (maybeSeed getOrElse s"minion${ThreadLocalRandom.current.nextInt}" getBytes StandardCharsets.UTF_8))
+    val seed          = maybeSeed getOrElse s"minion${ThreadLocalRandom.current.nextInt}"
```

Comment:
From the task: "Print the account seed. If it wasn't specified, **generate, print** and use it".
So this should be in WavesDexCli.

Reply:
[Generated](https://github.com/wavesplatform/matcher/pull/299/files#diff-6508a5494d9e6d57de82758561b8cc46R33), [printed](https://github.com/wavesplatform/matcher/pull/299/files#diff-dd5ce2fc18ee1b058b6aef14af9e1dcbR190) and [used](https://github.com/wavesplatform/matcher/pull/299/files#diff-dd5ce2fc18ee1b058b6aef14af9e1dcbR188)